UnitTesting/Repositories/ProductRepository/DeleteProductTests.cs

```diff
@@ -0,0 +1,56 @@
+﻿using Microsoft.EntityFrameworkCore;
+using NUnit.Framework;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace UnitTesting.Repositories.ProductRepository
+{
+    internal class DeleteProductTests : RepositoryTestsSetup
+    {
+        private readonly Guid productId = Guid.NewGuid();
+
+        [SetUp]
```

Comment:
use [OneTimeSetup] instead

Reply:
[SetUp] is needed but I have fixed productId initialization and moved it to Setup method.